internal/resources/ibm/is_share.go

```diff
@@ -0,0 +1,152 @@
+package ibm
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+// IsShare struct represents: A floating IP instance inside a VPC
+//
+// Resource information: https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/is_share
+// Pricing information: https://www.ibm.com/cloud/vpc/pricing
+type IsShare struct {
+	Address              string
+	Region               string
+	Zone                 string
+	Profile              string // general-purpose, 5iops-tier, 10iops-tier, custom, dp2 (plan for zonal file storage. tiered profiles are deprecated)
+	Size                 int64
+	IOPS                 int64 // optional maximum input/output per second, limited by size.
+	IsReplica            bool
+	InlineReplicaName    string
+	InlineReplicaZone    string
+	MonthlyInstanceHours *float64 `infracost_usage:"is-share_monthly_instance_hours"`
+	TransmittedGB        *int64   `infracost_usage:"is-share_monthly_transmitted_gb"`
+}
+
+// IsShareUsageSchema defines a list which represents the usage schema of IsShare.
+var IsShareUsageSchema = []*schema.UsageItem{
+	{Key: "monthly_instance_hours", DefaultValue: 730, ValueType: schema.Float64},
+	{Key: "monthly_transmitted_gb", DefaultValue: 0, ValueType: schema.Int64},
+}
```

Comment:
Should the usage variable name match in these 2 places? If so, we could leave it as `monthly_instance_hours` and `monthly_transmitted_gb` to remain consistent with the other resources

Reply:
yes, will fix thank you